components/src/banner/banner.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="tw-py-2.5 tw-px-4 tw-text-contrast tw-flex tw-gap-2" [ngClass]="bannerClass">
```

Comment:
I didn't know you could use decimals in Tailwind like this 😮  seems like too much choice to me! (your code is fine though)